scripts/tests/test_infra_k8s_action_flow.py

```
@@ -0,0 +1,161 @@
+    for key, value in inputs.items():
+        monkeypatch.setenv(key, value)
+
+    raw_values = {field: None for field in RawInputs.__annotations__}
+    raw_values["node_pools"] = "[]"
+    inputs_resolved = _resolve_all_inputs(RawInputs(**raw_values))
+    prepare_inputs(inputs_resolved)
+    _apply_env_file(monkeypatch, env_file)
+
+    monkeypatch.setattr(
+        "scripts.provision_cluster.tofu_init",
+        lambda *_args, **_kwargs: TofuResult(True, "", "", 0),
```

Ruff (0.14.13)

Boolean positional value in function call

(FBT003)